pkg/server/skydb/skydb.go

```diff
@@ -21,3 +21,14 @@ import (
 
 var log = logging.LoggerEntry("skydb")
 var timeNow = func() time.Time { return time.Now().UTC() }
+
+// SetTimeNowForTestingOnlyPleaseDoNotUseThis sets timeNow
+// for testing purpose only.
+func SetTimeNowForTestingOnlyPleaseDoNotUseThis(f func() time.Time) {
+	timeNow = f
+}
+
+// GetTimeNowForTestingOnlyPleaseDoNotUseThis returns timeNow
+func GetTimeNowForTestingOnlyPleaseDoNotUseThis() func() time.Time {
+	return timeNow
+}
```

Comment:
We should put this method into `skydb_test.go`

Reply:
I found it is impossible. I submit a new PR on reducing the number of public func: https://github.com/SkygearIO/skygear-server/pull/518